shared/fs/common/path-item-action/choose-view.js

```diff
@@ -10,8 +10,8 @@ import Confirm from './confirm-container'
 
 type OwnProps = {|
   floatingMenuProps: FloatingMenuProps,
+  mode: 'row' | 'screen',
   path: Types.Path,
-  position?: 'row' | 'header',
```

Comment:
@jakob223 I ended up changing the name into `mode` as well, as `mode` is also used in `path-item-info` and I thought this'd be more consistent.